modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer Operator
+Users that wish to install the AWS Load Balancer Operator (ALBO) can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing {product-title} cluster with BYO-VPC installed in STS mode
+* Cluster-admin access
+* Access to modify the VPC and subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone (LZ), the LZ is enabled for the account and the AWS ALBO is an available feature within the LZ. Additionally, the ability to use Amazon Elastic Load Balancing (ELB) V2 with {product-title} is subject to the availability of Application Load Balancers (ALBs) and Network Load Balancers (NLBs) in availability zones (AZ) and the AWS LZ.
+====
+[IMPORTANT]
```

Comment:
Covers row 43 in https://docs.google.com/spreadsheets/d/1nwUdersMAaKZ4AUX_KhCpgsjL6DByxVy2B95CNqaeYY/edit#gid=1241421272